pgdrive/scene_creator/vehicle/base_vehicle.py

```diff
@@ -372,13 +372,12 @@ def update_dist_to_left_right(self):
         self.dist_to_left, self.dist_to_right = self._dist_to_route_left_right()
 
     def _dist_to_route_left_right(self):
-        current_reference_lane = self.routing_localization.current_ref_lanes[-1]
+        current_reference_lane = self.routing_localization.current_ref_lanes[0]
```

Comment:
What about add a property to BaseVehicle: v.reference_lane

Reply:
Ok, I will add a property to access it quickly. And it will be self.routing_localization.current_ref_lanes[0]

Reply:
Humm, I think return self.routing_localization.current_ref_lanes will be better